data/pathfinder/paizo/roleplaying_game/core_rulebook/cr_feats.lst

```diff
@@ -166,7 +166,7 @@ Step Up											CATEGORY:FEAT		TYPE:Combat
+Stunning Fist										CATEGORY:FEAT		TYPE:Combat.AttackOption			PREABILITY:1,CATEGORY=FEAT,Improved Unarmed Strike																																																										PRESTAT:2,DEX=13,WIS=13	PRETOTALAB:8									DEFINE:StunningFistMonkLVL|0	DEFINE:StunningFistAttack|0	DEFINE:StunningFistDC|0											DESC:You know just where to strike to temporarily stun a foe. %1/day (DC %2)|StunningFistAttack|StunningFistDC																					STACK:NO																																																		ABILITY:Ability Focus|AUTOMATIC|Stunning Fist																																																																																																																																																				BONUS:VAR|StunningFistDC|2|PREABILITY:1,CATEGORY=FEAT,Ability Focus (Stunning Fist)	BONUS:VAR|StunningFistDC|10+(TL/2)+WIS	BONUS:VAR|StunningFistAttack|floor(MonkLVL+(MonkLVL/4))|TYPE=NoStack															SOURCEPAGE:p.135	BENEFIT:You must declare that you are using this feat before you make your attack roll (thus, a failed attack roll ruins the attempt). Stunning Fist forces a foe damaged by your unarmed attack to make a Fortitude saving throw (DC %1), in addition to dealing damage normally. A defender who fails this saving throw is stunned for 1 round (until just before your next turn). A stunned character drops everything held, can't take actions, loses any Dexterity bonus to AC, and takes a -2 penalty to AC. You may attempt a stunning attack once per day for every four levels you have attained (but see Special), and no more than once per round. Constructs, oozes, plants, undead, incorporeal creatures, and creatures immune to critical hits cannot be stunned.|StunningFistDC
```

Comment:
I tried to implement what you suggested @LegacyKing but I didn't have any success. I reverted back and started trying to understand what you said better. 

Looking further into the issue it looks like it's on the Unchained Monk side, Standard Monk in my tests seems to give the right number of uses per day, even paired with a non-monk class.

Unchained Monk is giving stunning fist uses equal to (it looks like) 1+TL/4. Which I'm guessing means it's leaving monk levels stuck at 1 when applying Unchained Monk which doesn't make sense because things like BAB and saves are changing the way they're supposed to and they're based on MonkLVL also. 

I'm going to go back to looking and testing, just wanted to update.